### ISSUE TITLE
Attempted to Patch ROM, Does not Get Past Initial Boy/Girl Screen

### ISSUE THREAD

**Issue by echondropoulos** [ArthropodAssault/Shining-Crystal-Patch-and-Documentation#1]
Hello,

I have attempted to use both patches(IPS and UPS) on a two different Crystal ROMs, but they seem to get stuck on a blank white screen after Prof. Oak appears for the boy/girl question.

Any ideas? What is the signature on the ROM you used as the base?

ROMs I've tried:

* MD5: 9f2922b235a5eeb78d65594e82ef5dde
* MD5: 301899b8087289a6436b0a241fbbb474

Thanks!

**Comment by ArthropodAssault** [comment]
Hi, 

Thank you for the feedback. I don't have experience with md5sum so don't know how to check the signature quickly. I can take some time and figure it out. I tried patching the ROMs from the Reddit ROMs megathread and encountered the same issue as you. I created new IPS and UPS patches from the vanilla Crystal ROMs available on the megathread and the patches are now working with those ROMs. I've pushed them to the Shining Crystal Github, so hopefully this solves your issue. I will check on the MD5 signature of the original vanilla ROM I used later, although it may be mostly a moot point. 

Thanks again.
